esphome/esp32-d1-mini.yaml

```diff
@@ -51,14 +54,19 @@ uart:
   id: uart_bus
   tx_pin: GPIO1
```

Comment:
I had to switch places between the rx and tx pins to make it work. Found the information here - https://github.com/esphome/issues/issues/4317#issuecomment-1474937332.

Reply:
Hmm, that's odd. Thanks for raising this. I haven't tried this with an ESP32 D1 Mini, but it works as expected with my C3 Mini. I'll try this a D1 Mini as well for good measure.